adam-core/src/main/scala/org/bdgenomics/adam/util/FileExtensions.scala

```diff
@@ -112,10 +112,35 @@ object FileExtensions {
    * @param pathName The path name to match.
    * @return Returns true if the path name matches a VCF format file extension.
    */
-  private[adam] def isVcfExt(pathName: String): Boolean = {
+  def isVcfExt(pathName: String): Boolean = {
     pathName.endsWith(".vcf") ||
       pathName.endsWith(".vcf.gz") ||
       pathName.endsWith(".vcf.bgzf") ||
       pathName.endsWith(".vcf.bgz")
   }
+
+  /**
+   * @param pathName The path name to match.
+   * @return Returns true if the path name matches an HTSJDK sequence dictionary (.dict) extension.
+   */
+  def isDictExt(pathName: String): Boolean = {
+    pathName.endsWith(".dict")
+  }
+
+  /**
+   * @param pathName The path name to match.
+   * @return Returns true if the path name matches a Bedtools genome file (.genome) extension.
+   */
+  def isGenomeExt(pathName: String): Boolean = {
+    pathName.endsWith(".genome") ||
+      pathName.endsWith(".genome.txt")
```

Comment:
All three (`.genome`, `.genome.txt`, and `.txt`) go through the same code path.  The only difference is the log message.